airbyte-integrations/connectors/source-close-com/source_close_com/offset_increment_workaround.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import InitVar, dataclass
+from typing import Any, List, Mapping, Optional, Union
+
+import requests
+from airbyte_cdk.sources.declarative.interpolation import InterpolatedString
+from airbyte_cdk.sources.declarative.requesters.paginators.strategies.pagination_strategy import PaginationStrategy
+from airbyte_cdk.sources.declarative.types import Config
+from dataclasses_jsonschema import JsonSchemaMixin
+
+
+@dataclass
+class OffsetIncrementWorkaround(PaginationStrategy, JsonSchemaMixin):
```

Comment:
could you please add a docstring explaining the difference between this class and other paginating strategies?